From b3921b26e8659ec08476b070e4547e6eddc46781 Mon Sep 17 00:00:00 2001
From: Claudiu Belu <cbelu@cloudbasesolutions.com>
Date: Tue, 27 Sep 2016 06:46:32 -0700
Subject: [PATCH] Installs futures only for python 2.7 and 2.6

The futures library is native to python 3 and it doesn't
have to be installed from pypi.

If the futures library is installed, it can cause issues when used.

Changes the futures' library requirement to match the one in
global requirements. [1]

[1] https://github.com/openstack/requirements/blob/master/global-requirements.txt

Change-Id: I8f13d63a303b71f7bdd8c3e67c15d0a0df5ea7a9
Closes-Bug: #1628107
---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 88b3df2..6d31e09 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
-futures>=2.1.3
+futures>=3.0;python_version=='2.7' or python_version=='2.6' # BSD
 requests>=1.1
 six>=1.5.2
